src/sql/src/postgres_util.rs

```diff
@@ -28,31 +28,26 @@ pub struct PgColumn {
 ///
 /// - Invalid connection string, user information, or user permissions.
 /// - Upstream table does not exist or contains invalid values.
-pub async fn fetch_columns(
-    conn: &str,
-    namespace: &str,
-    table: &str,
-) -> Result<Vec<PgColumn>, anyhow::Error> {
+pub async fn fetch_columns(conn: &str, table: &str) -> Result<Vec<PgColumn>, anyhow::Error> {
     let (client, connection) = tokio_postgres::connect(&conn, NoTls).await?;
     tokio::spawn(connection);
 
+    // Strip table name of quotation marks, if they exist.
+    let query = format!(
+        "SELECT '{}'::regclass::oid",
+        table
+            .strip_prefix("\"")
+            .unwrap_or_else(|| &table)
+            .strip_suffix("\"")
+            .unwrap_or_else(|| &table)
```

Comment:
Ah, yeah, eek, this is going to mishandle table names that have single quotes in them, and possible identifiers with other sorts of special characters. I think the original query was definitely better here—sorry!

Reply:
Ok, I think I see what's going on here. The idea with using unresolved names is that you'd no longer need the `NAMESPACE` option in `CREATE SOURCE(S)`. Instead the namespace should come from the part before the dot; so

```
CREATE SOURCE blah FROM POSTGRES HOST 'foo:5432' TABLES (
    public.one AS renamed
    public.two AS renamed2
    weird_schema.three
)
```

So then here you'd require that `table` be an object name of two components, and pass the first component into the original query as the namespace and the second component into the original query as the table name.